### ISSUE TITLE
Reach out to specific members of the community to do a review

### ISSUE THREAD

**Issue by marcoscaceres** [ResponsiveImagesCG/ri-usecases#7]
@Wilto could you please reach out to key participants of the CG for a review of the UC&Reqs doc? We particularly need help with the accessibility and API design requirements (i.e., have they been sufficiently articulated or are we missing something?).

**Comment by marcoscaceres** [comment]
We've done outreach to both the mailing list and on twitter.  We had a high reach of about 50,000 people on twitter. The following data was gathered from google analytics: 

76 people visited this site
Visits: 85
Unique Visitors: 76
Pageviews: 97
Pages / Visit: 1.14
Avg. Visit Duration: 00:00:44
Bounce Rate: 90.59%
% New Visits: 89.41%
89.41%New Visitor
76 Visits
10.59%Returning Visitor
9 Visits

Lesson learned: we might need to do direct 1 to 1 outreach to particular people in the community to do a review.
